docker-compose.yml

```diff
@@ -87,6 +87,8 @@ services:
       - ./htmlcov:/opt/warehouse/src/htmlcov:z
       - ./setup.cfg:/opt/warehouse/src/setup.cfg:z
       - ./pyproject.toml:/opt/warehouse/src/pyproject.toml:z
+      - ./.coveragerc:/opt/warehouse/src/.coveragerc:z
```

Comment:
Moving the contents of `.coveragerc` into `pyproject.toml` should avoid the need for this mapping 🙂